src/components/EmojiPicker/EmojiPickerMenu/EmojiPickerMenu.jsx

```diff
@@ -499,7 +496,7 @@ const EmojiPickerMenu = (props) => {
             </View>
             {!isFiltered && (
                 <CategoryShortcutBar
-                    headerEmojis={headerEmojis.current}
+                    headerEmojis={headerEmojis}
```

Comment:
Previously, in our class component `headerEmojis` was a ref. and we should not read `ref.current` during rendering.

So i converted `headerEmoji` ref to a state instead.

   - Docs reference 
![image](https://github.com/Expensify/App/assets/60546840/42f9eaa6-b4cc-4cb0-b438-bf4ec23fd695)